src/message.coffee

```diff
@@ -169,6 +185,23 @@ class SlackTextMessage extends TextMessage
       .then (substrings) ->
         return substrings.join("")
 
+  ###*
+  # Fetch conversation information from conversation map
+  # @private
+  ###
+  fetchConversation: (client, conversationId) ->
```

Comment:
can this just be a method inside `client`? it feels a little strange to me that this message instance is responsible for reaching inside the `client` and updating its state (the `channelData` map).